src/main/api/2/studio-api-2.yaml

```diff
@@ -3488,6 +3488,52 @@ paths:
         '403':
           $ref: '#/components/responses/Forbidden'
 
+  /configuration/content-type/preview_image:
+    get:
+      tags:
+        - configuration
+      summary: Get preview image of a given content-type
```

Comment:
Get preview image of a given content type

Reply:
Updated. Thank you!